dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+
+  SOUTH:
+    'AD': [Andorra, Andorran*, Andorrense*, Andorrian*, Andorrenh*]
+    'AL': [Albânia, Albanês*, Tirana]
+    'BA': [Bósnia e Herzegovina, Bósni*, Sarajevo, Saraievo]
+    'ES': [Espanha, Espanhol*, Espanhóis, Madrid, Madri, Barcelona]
+    'GI': [Gibraltar, Gibraltarin*, Gibraltin*, Llanit*, Llanis*]
+    'GR': [Grécia, República Helênica, República Helénica, Grego*, Grega*, Atenas]
+    'HR': [Croácia, Croata*, Zagreb, Zagrebe]
+    'IT': [Itália, República Italiana, Italian*, Roma]
+    'KV': [Kosovo, Cossovo, Cosovo, Kossovo, Kosovar*, Cosovar*, Cossovar*, Pristina]
+    'ME': [Montenegro, Montenegrin*, Podgoritza, Podgoriza, Podgorica]
+    'MK': [Macedônia do Norte, Macedónia do Norte, Macedóni*, Escópia, Skopje, Skopie]
+    'MT': [Malta, Maltês*, Valeta]
+    'PT': [Portugal, República Portuguesa, Português*, Portugues*, Lisboa]
+    'RS': [Sérvia, Sérvi*, Belgrado]
+    'SI': [Eslovénia, Eslovênia, Esloven*, Liubliana, Ljubljana]
+    'SM': [San Marino, São Marinho, São Marino, Samarinês*, São-Marinense*, São-Marinhense*, San-Marinense*]
+    'VA': [Vaticano, Vatican*]
+
+  WEST:
+    'AT': [Áustria, Austríac*, Viena]
+    'BE': [Bélgica, Belga*, Bélgic*, Bruxelas]
+    'CH': [Suíça, Suíç*, Zurique, Berna]
+    'DE': [Alemanha, Alemã*, Berlim, Frankfurt, Francoforte]
+    'FR': [França, República Francesa, Francês*, Frances*, Paris]
+    'LI': [Liechtenstein, Listenstaine, Liechtensteinense*, Liechtensteiniense*, Liechtensteinian*, Listenstainian*, Listenstainês*, Vaduz]
+    'LU': [Luxemburgo, Luxemburguês*]
+    'MC': [Mónaco, Mônaco, Monegasc*]
+    'NL': [Países Baixos, Holanda, Neerlandês*, Holandês*, Amesterdão, Amsterdão, Amsterdã, Amesterdã]
+
+OCEANIA:
+  AU-NZ:
+    'AU': [Austrália, Australian*, Aussie*, Oz, Camberra, Sydney]
+    'CK': [Ilhas Cook, Cookian*, Cookense*, Avarua]
+    'NF': [Ilha Norfolk, Ilha Norfolque, Norfolquin*]
+    'NZ': [Nova Zelândia, Neozelandês*, Kiwi*, Wellington, Auckland, Auclanda]
+
+  MEL:
+    'FJ': [Fiji, Fíji, Fidji, Fijian*]
+    'NC': [Nova Caledónia, Nova Caledônia, Neocaledóni*, Neocaledôni*, Numeá, Numea, Numeia]
+    'PG': [Papua-Nova Guiné, Papua Nova Guiné, Papuási*, Porto Moresby, Port Moresby]
+    'SB': [Ilhas Salomão, Salomónic*, Salomônic*, Salomonense*, Honiara]
+    'VU': [Vanuatu, Vanuatuense*, Porto Vila]
+
+  MIC:
+    'FM': [Estados Federados da Micronésia, EFM, Micronési*, Paliquir]
+    'GU': [Guam, Guão, Guame, Guamês*, Guames*, Aganha]
+    'KI': [Kiribati, Quiribáti, Quiribati, Kiribatian*, Etemaquian*, Kiribatin*, Taraua, Taraua do Sul, Tarawa do Sul]
+    'MH': [Ilhas Marshall, Marshallin*, Marshalês*, Majuro]
+    'MP': [Ilhas Marianas Setentrionais, Ilhas Marianas do Norte, Norte-Marianense*, Capital Hill, Saipã, Saipan]
+    'NR': [Nauru, Nauruan*, Nauruense*, Yaren, Iarém]
+    'PW': [Palau, Palauan*, Palauense*, Melekeok, Melequeoque]
+
+  POL:
+    'AS': [Samoa Americana, Estadunidense*, Samoan*, Pago Pago]
```

Comment:
should the , after Estadunidense* be removed? Otherwise it will match US and Samoan. Might create false matches.

Reply:
I agree that it could lead to false matches. I found it as a demonyn but I wasn't sure if I should keep it. I believe that it might be best to just remove it.